app/assets/stylesheets/caffeine/admin/widgets/medium.css.scss

```diff
@@ -4,18 +4,8 @@
 @import "caffeine/admin/theme/mixins";
 @import "caffeine/admin/includes/functions";
 
-$buttons: (
-  bold: $fa-var-bold,
-  italic: $fa-var-italic,
-  underline: $fa-var-underline,
-  anchor: $fa-var-link,
-  quote: $fa-var-quote-left,
-  unorderedlist: $fa-var-list-ul,
-  orderedlist: $fa-var-list-ol,
-  justifyLeft: $fa-var-align-left,
-  justifyCenter: $fa-var-align-center,
-  justifyRight: $fa-var-align-right
-);
+$padding: 0.25em;
+$buttons-last: 'underline', 'quote', 'h3','orderedlist';
```

Comment:
Commas in lists should be followed by a single space<br>Prefer double-quoted strings